assets/scss/components/_breadcrumb.scss

```
@@ -63,3 +61,29 @@
         }
     }
 }
+
+.breadcrumb-alt {
+    @extend .breadcrumb;
+    color:white;
+
+    @media only screen and #{$media-wide} {
+        background:#225871;
+
+        &:after {
+            background-image:none;
+        }
+
+        ol {
+            li {
+                a {
+                    color: #ddd;
+                }
+
+                &:not(:last-child):after {
+                    @include sprite();
+                    @include sprite-position($ariane-alt);
+                }
+            }
+        }
```

Concaténation/Factorisation
```suggestion
        ol li {
            a {
                color: #ddd;
            }

            &:not(:last-child):after {
                @include sprite();
                @include sprite-position($ariane-alt);
            }
        }
```